test/test_data/openssl.py

```diff
@@ -20,17 +26,16 @@
         "product": "openssl",
         "version": "1.0.2g",
     },
-    {
-        "url": "https://files.pythonhosted.org/packages/ba/91/84a29d6a27fd6dfc21f475704c4d2053d58ed7a4033c2b0ce1b4ca4d03d9/",
-        "package_name": "cryptography-3.0-cp35-abi3-manylinux2010_x86_64.whl",
-        "product": "openssl",
-        "version": "1.1.1g",
-        "other_products": ["gcc"],
-    },
```

Comment:
The cryptography package really does include a copy of openssl.  Or it did at the time this test was added, anyhow. I don't know offhand if later versions found another way, but it was policy to include the correct, latest openssl as part of the wheel to reduce dependency friction with various OSes.

So I think we should keep this test.  It's surprising but likely useful to make sure we don't stop detecting that.

Reply:
I updated the pattern to continue to detect openssl in cryptography